src/sparseml/exporters/transforms/conv_to_convinteger_add_cast_mul.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from onnx import ModelProto
+
+from sparseml.exporters.transforms import OnnxTransform
+from sparseml.exporters.transforms.utils import (
+    INITIALIZER_MATCH,
+    MatchResult,
+    add_quantized_conv_matmul_add_ops,
+    get_quantization_params,
+    get_structural_matches,
+)
+from sparseml.onnx.utils import ONNXGraph
+
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["ConvToConvIntegerAddCastMul"]
```

Comment:
can we rename `AddCastMul` to something denoting 'bias add' or 'scaled bias add'